replacements.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
maybe call this something like TestRequestReplacements.json or something for clarity?

Reply:
it comes from [https://github.com/PlayFab/SDKGenerator/blob/master/targets/postman/replacements.json](https://github.com/PlayFab/SDKGenerator/blob/master/targets/postman/replacements.json)

not sure if its worth renaming.  i can.  what do you think?

---
In reply to: [631952963](https://github.com/jplafonta/XPlatCTemplate/pull/16#discussion_r631952963) [](ancestors = 631952963)

Reply:
ah I guess its fine as is then